website/content/docs/connect/cluster-peering/index.mdx

```diff
@@ -40,6 +40,7 @@ Regardless of whether you connect your clusters through WAN federation or cluste
 | Gossip protocol: Requires LAN gossip only          |    &#10060;    |    &#9989;      |
 | Forwards service requests for service discovery    |    &#9989;     |    &#10060;     |
 | Shares key/value stores                            |    &#9989;     |    &#10060;     |
+| Shares ACL tokens                                  |    &#9989;     |    &#10060;     |
```

Comment:
Not sure if this is technically true, but it seems to explain to an audience like myself why the process is so different for peering, and you don't need to transfer ACLs between clusters.